src/facet.scss

```
@@ -0,0 +1,71 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-facet;
+
+.#{$block} {
+  @include fd-reset();
+
+  &-group {
+    @include fd-reset();
+
+    @include fd-flex() {
+      flex-wrap: wrap;
+      align-items: flex-start;
+    }
+  }
+
+  &--image {
+    @include fd-flex();
+
+    // when used in object/dynamic page header title, we need different alignment
+    &-header-title {
+      align-self: flex-end;
+    }
+  }
+
+  &--form {
+    @include fd-flex(column);
+  }
+
+  &--key-value {
+    .#{$block}__object-status {
+      .#{$block}__object-status-text {
+        font-family: var(--sapFontFamily);
+      }
+    }
```

Can this be simplified to
```suggestion
  &--key-value {
      .#{$block}__object-status-text {
        font-family: var(--sapFontFamily);
      }
```
?

---

Unfortunately, no. I had tried this, but object status component takes higher specificity and its `font-family` gets applied instead.